authcodepkce.html

```diff
@@ -0,0 +1,332 @@
+﻿<!DOCTYPE html>
+
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>OAuth Example</title>
+</head>
+
+<body>
+    <div class="container">
+        <h1>Digital DJ Pool OAuth Example: Auth Code PKCE Client</h1>
+        <h2>Client Application Authenticated as <span id="spanUsername"></span> </h2>
+        <fieldset>
+            <legend>Authorization Request</legend>
+            <p>
+                <input id="btnAuthorize" value="Authorize and get new Access Token" type="submit" />
+            </p>
+        </fieldset>
+        <fieldset>
+            <legend>Tokens</legend>
+            <p>
+                Access Token
+                <input id="txtAccessToken" width="604" type="text" readonly />
+                <input id="btnRefresh" value="Get new access_token with Silent Authentication" type="submit" />
+            </p>
+            <p>
+                Refresh Token
+                <input id="txtRefreshToken" width="604" type="text" readonly />
+            </p>
+            <p>
+                Log:
+                <pre id="divAuthActivity">
+        </pre>
+            </p>
+            <input id="btnClearTokens" value="Clear Tokens" type="submit" />
+        </fieldset>
+        <fieldset>
+            <legend>APIs</legend>
+            <h5>Get an user's crates</h5>
+
+            <p>
+                <input id="txtGetCratesFor" value="ApiDemoUser" type="text" />
+                <input id="btnGetCratesFor" value="Call DJP API" type="submit" />
+            </p>
+            <h5>Get crate details</h5>
+            <p>
+                <input id="txtCrateId" value="93089" type="text" />
+                <input id="btnCallApi" value="Call DJP API" type="submit" />
+            </p>
+
+            <p>
+                Output:
+                <pre id="divApiResponse">
+      </pre>
+            </p>
+        </fieldset>
+    </div>
+    <script type="text/javascript">
+
+        /*********************** Variables ******************************************/
+
+        const client_id = 'YOUR_CLIENT_ID_HERE'; // the client id of the client application
+
+        /*
+         * the client provided redirect uri on the client host. This is required
+         * Change this with the page that will handle the redirection from digitaldjpool.com
+         */
+        const redirect_uri = 'http://localhost:3000/authcodepkce.html';
```

Comment:
The other examples are using port 8080 from 'live-server-https' module as documented in the readme. Any issue using the existing approach?

Reply:
no issue. i just didn't see the readme.  I removed the server.js.